### DISCUSSION TITLE
Is it possible to ignore changes in "with-baseline" mode?

### DISCUSSION THREAD

**Discussion by theagilepilot** [stryker-mutator/stryker-net#2870] [Q&A] [answered]
I am creating a baseline for the 'develop' version on my code, it runs, and then it creates a baseline file. When I run again, immediately after that, it detects "non-csharp files were changed" and runs against the whole code.

Is it possible to ignore the baseline file as a change like the "since.ignore-changes-in" feature?

**Comment by rouke-broersma** [comment, marked as answer]
Yep, you use since ignore changes ;)
